pkg/kn/commands/revision_describe.go

```diff
@@ -16,10 +16,11 @@ package commands
 
 import (
 	"errors"
+	"github.com/knative/client/pkg/serving"
```

Comment:
in order to have distinction between kn serving and actual serving lib, lets streamline the alias for one of them?

For eg: we alias `servinglib` for kn serving pkg, while we use the knative/serving as is.

WDYT ?